test/suites/cli/main_test.go

```diff
@@ -35,7 +35,12 @@ func setup() (teardown func(), err error) {
 
 	teardown = func() {
 		log.Println("[TEARDOWN]")
-		utils.SnapRemove(nil, cliSnap)
+
+		if utils.SkipTeardownRemoval {
+			log.Println("[SKIP] Removal of snaps in teardown is skipped.")
```

Comment:
`[CAPS]` is used to log the testing staged such as `[SETUP]` and `[TEARDOWN]`. It is misused here for a simple log message that is inside the teardown stage.

Reply:
This comment applies to all such messages.

Reply:
The reason I utilized [SKIP] to emulate the output of `t.skip`, so that it appears similar to: `--- SKIP: TestTLSCert (0.00s)`

Reply:
The reason I utilized [SKIP] to emulate the output of `t.skip`, so that it appears similar to: `--- SKIP: TestTLSCert (0.00s)`